staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy.go

```diff
@@ -56,11 +64,17 @@ type proxyHandler struct {
 }
 
 func (r *proxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
-	proxyRoundTripper, err := r.getRoundTripper()
-	if err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+	handlingInfo := r.handlingInfo.Load().(proxyHandlingInfo)
+	if handlingInfo.local {
+		r.localDelegate.ServeHTTP(w, req)
 		return
 	}
+
+	if handlingInfo.transportBuildingError != nil {
```

Comment:
> is this the case where they registered a service with a bad CA bundle or something? not sure I'd expect that the details of that error to bubble to an external user

Don't plan to change that in this pull.  I considered hiding it, but then I couldn't come up with a way for a cluster-admin to debug it.